boards/arm/pico_pi_m4/board.h

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
can you remove board.h it shouldn't be needed.

Reply:
Ok i'm doing it